spec/src/main/asciidoc/WebServicesMetadata.adoc

```diff
@@ -943,2259 +807,1399 @@ header, and the mode is OUT.
 
-a|
-targetNamespace
+==== RPC Operation Style
 
-a|
-The XML namespace for the return value.
+In the RPC operation style, the parameters and return values map to
+separate parts on the WSDL input and output messages. The @WebParam.mode
+annotation determines the messages in which a particular parameter
+appears. IN parameters appear as parts in the input message, OUT
+parameters appear as parts in the output message, and INOUT parameters
+appear as parts in both messages. The order of parameters in the method
+signature determines the order of the parts in the input and output
+message. The return value is the first part in the output message.
 
-a|
-The empty namespace, if the operation is document style, the
+In the rpc/literal operation mode, each message part refers to a
+concrete schema type. The schema type is derived from the Java type for
+the parameter, as described in section 5 - Java Mapping To XML/WSDL.
 
-| a|
-Only used if the operation is
+==== Document Operation Style
 
-a|
-parameter style is
+In the document operation style, the input and output WSDL messages have
+a single part referencing a schema element that defines the entire body.
+Jakarta Web Services Metadata implementations MUST support both the “wrapped” and “bare”
+styles of document / literal operation. The developer may specify which
+of these styles is in effect for a particular operation by using the
+@SOAPBinding.parameterStyle annotation.
 
-| a|
-document style or the return value
+==== Document “Wrapped” Style
 
-a|
-WRAPPED, and the
+In the “wrapped” operation style, the input and output messages contain
+a single part which refers (through the _element_ attribute) to a global
+element declaration (the _wrapper_)
 
-| a|
-maps to a header.
+of complexType defined using the xsd:sequence compositor. The global
+element declaration for the input message has a local name equal to
+@WebMethod.operationName. The global element declaration for the output
+message (if it exists) has a local name equal to
+@WebMethod.operationName + “Response”. Both global element declarations
+appear in the @WebService.targetNamespace.
 
-If the target namespace is set to “”, this represents the empty
+Non-header method parameters and return values map to child elements of
+the global element declarations defined for the method. The order of
+parameters in the parameter list determines the order in which the
+equivalent child elements appear in the operation’s global element
+declarations.
 
-a|
-return value does not map to a header,
+The @WebParam.name and @WebParam.targetNamespace annotation elements
+determine the QName of a parameter’s child element, while the
+@WebResult.name and @WebResult.targetNamespace annotations determines
+the QName of the return value’s child element. The schema type for each
+child element is derived from the type of the Java parameter or return
+value, as described in section _5 Java Mapping To XML/WSDL._
```

Comment:
"section 5..." can be a link

Reply:
ok, resolved by https://github.com/eclipse-ee4j/jws-api/pull/69/commits/b03628ddda22c1bad0221dec7bfd3dc9de6788f8